website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,506 @@
+---
+layout: "docs"
+page_title: "Helm - Kubernetes"
+sidebar_current: "docs-platform-k8s-helm"
+description: |-
+  The Vault Helm chart is the recommended way to install and configure Vault on Kubernetes.
+---
+
+# Helm Chart
+
+The [Vault Helm chart](https://github.com/hashicorp/vault-helm)
+is the recommended way to install and configure Vault on Kubernetes.
+In addition to running Vault itself, the Helm chart is the primary
+method for installing and configuring Vault to integrate with other
+services such as Consul for High Availability deployments.
+
+This page assumes general knowledge of [Helm](https://helm.sh/) and
+how to use it. Using Helm to install Vault will require that Helm is
+properly installed and configured with your Kubernetes cluster.
+
+-> **Important:** The Helm chart is new and
+may still change significantly over time. Please always run Helm with
+`--dry-run` before any install or upgrade to verify changes.
+
+~> **Security Warning:** By default, the chart will install an insecure configuration
+of Vault. This provides a less complicated out-of-box experience for new users,
+but is not appropriate for a production setup. It is highly recommended to use
+a [properly secured Kubernetes cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/). 
+See the [architecture reference](/docs/platform/k8s/run.html#architecture) 
+for a Vault Helm production deployment checklist.
+
+## Using the Helm Chart
+
+To use the Helm chart, you must download or clone the
+[vault-helm GitHub repository](https://github.com/hashicorp/vault-helm)
+and run Helm against the directory. We plan to transition to using a real
+Helm repository soon. When running Helm, we highly recommend you always
+checkout a specific tagged release of the chart to avoid any
+instabilities from master.
+
+Prior to this, you must have Helm installed and configured both in your
+Kubernetes cluster and locally on your machine. The steps to do this are
+out of the scope of this document, please read the
+[Helm documentation](https://helm.sh/) for more information.
+
+Example chart usage:
+
+```sh
+# Clone the chart repo
+$ git clone https://github.com/hashicorp/vault-helm.git
+$ cd vault-helm
+
+# Checkout a tagged version
+$ git checkout v0.1.0
+
+# Run Helm
+$ helm install --dry-run ./
+```
+
+## Configuration (Values)
+
+The chart is highly customizable using
+[Helm configuration values](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing).
+Each value has a sane default tuned for an optimal getting started experience
+with Vault. Before going into production, please review the parameters below
+and consider if they're appropriate for your deployment.
+
+* <a name="v-global" href="#v-global">`global`</a> - These global values affect multiple components of the chart.
```

Comment:
Are these anchors required? I thought the site generator automatically created them? e.g. https://www.vaultproject.io/api/auth/jwt/index.html#oidc_discovery_url (the md has nothing special to achieve this).

You can test this by reviewing the preview build on netlify.